In-house patch to remove the ">> open console" link from instance
elements in the Network Topology screen.  This patch is
Solaris-specific and not suitable for upstream contribution.

--- horizon-2013.2.3/openstack_dashboard/dashboards/project/network_topology/templates/network_topology/client_side/_balloon_container.html.orig    2014-06-20 08:04:57.394535111 -0600
+++ horizon-2013.2.3/openstack_dashboard/dashboards/project/network_topology/templates/network_topology/client_side/_balloon_container.html     2014-06-20 08:05:47.621066190 -0600
@@ -16,7 +16,6 @@
       <div class="cell link">
         <a href="[[url]]">» view [[type]] details</a>
         [[#console_id]]
-        <a href="[[url]][[console]]" class="vnc_window">» open console</a>
         [[/console_id]]
       </div>
       <div class="cell delete">

